examples/helloworld/main.go

```diff
@@ -86,8 +85,10 @@ func runController(ctx context.Context, controllerContext *controllercmd.Control
 	agentAddon, err := addonfactory.NewAgentAddonFactory(addonName, fs, "manifests/templates").
 		WithGetValuesFuncs(getValues, addonfactory.GetValuesFromAddonAnnotation).
 		WithAgentRegistrationOption(registrationOption).
-		WithInstallStrategy(addonagent.InstallAllStrategy(agent.HelloworldAgentInstallationNamespace)).
-		BuildTemplateAgentAddon()
+		// WithInstallStrategy(addonagent.InstallAllStrategy(agent.HelloworldAgentInstallationNamespace)).
+		// BuildTemplateAgentAddon()
+		WithManagementDir("manifests/management").
```

Comment:
does it mean we need to copy the same manifests in both management and hosted dir？

Reply:
No, only management manifests files should be held in the management dir.